imagekitio/url.py

```diff
@@ -211,3 +212,20 @@ def transformation_to_str(transformation):
             )
 
         return Default.CHAIN_TRANSFORM_DELIMITER.value.join(parsed_transforms)
+
+    @staticmethod
+    def custom_encodeURIComponent(url_str):
+        parsed_url = urlparse(url_str)
+        encoded_url = f"{parsed_url.scheme}://{parsed_url.netloc}"
```

Comment:
What is this statement for?
We are redefining the value of encoded_url in the next line.